labelbox/schema/resource_tag.py

```diff
@@ -0,0 +1,15 @@
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field, Relationship
+
+class ResourceTag(DbObject):
```

Comment:
Are these objects deleteable?

Reply:
not through the SDK for now. do I still need to add those